com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -59,21 +59,84 @@ public class DynamicResolutionHandler
         /// </summary>
         public Vector2Int finalViewport { get; set; }
 
-
         private DynamicResolutionType type;
 
         private PerformDynamicRes m_DynamicResMethod = null;
-        private static DynamicResolutionHandler s_Instance = new DynamicResolutionHandler();
+
+        private GlobalDynamicResolutionSettings m_CachedSettings = GlobalDynamicResolutionSettings.NewDefault();
+
+        private WeakReference m_OwnerCameraWeakRef = null;
+        private static Dictionary<int, DynamicResolutionHandler> s_CameraInstances = new Dictionary<int, DynamicResolutionHandler>();
+        private static DynamicResolutionHandler s_DefaultInstance = new DynamicResolutionHandler()
+        {
+            m_DynamicResMethod = DefaultDynamicResMethod
```

Comment:
Any specific reason why to move this out of the constructor?

Reply:
Yes. Default behaviour. 
If you look below, a user must set the dynamic res method explicitely per camera. Id not, then the default one (this one) is used.
If we dont take this approach, it means that the user would require to revise their code, and ensure they bind the camera they want the method to be on.
An alternative is to take this out of the instance i guess..

Reply:
I simplified this a lot by making the method and scaler type statics.